src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -1944,22 +2063,48 @@ public int getWaterUndergroundCreatureSpawnLimit()
 	@Override
 	public int getMaxWorldSize()
```

Comment:
When worlds are created, the world border size should be set to this by default.

Reply:
Since this is uses Methods in WorldBorder that are not yet implemented, I would split this into a new PR